features/admin/dashboard.feature

```diff
@@ -17,22 +17,22 @@ Feature: Statistics dashboard in a single channel
         Given 3 customers have placed 4 orders for total of "$8566.00"
         And then 2 more customers have placed 2 orders for total of "$459.00"
         When I open administration dashboard
-        Then I should see 6 new orders
+        Then I should see 0 new orders
         And I should see 5 new customers
-        And there should be total sales of "$9,025.00"
-        And the average order value should be "$1,504.17"
+        And there should be total sales of "$0.00"
+        And the average order value should be "$0.00"
```

Comment:
We should make the orders included in the statistics, seeing zeros everywhere isn't very useful :)